### PR TITLE
feat: #21826 Project Details header icons and design popup done

### PR DESCRIPTION
**Pull request by harpreetwalia143**

### CODE REVIEW

File: src/app/manage-learn/project/learning-resources/learning-resources.page.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!-- <ion-header>
+  <app-header
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/learning-resources/learning-resources.page.html:2]: remove this commented part

File: src/app/manage-learn/project/learning-resources/learning-resources.page.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    [bottomBorder]="true"
+    [showMenu]="false"
+    [showReload]="false"
+    [showSearch]="false"
+    [hideNotification]="true"
+  >
+    <ion-back-button></ion-back-button>
+  </app-header>
+</ion-header> -->
+
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/learning-resources/learning-resources.page.html:13]: Where is the ion-app tag outside??

File: src/app/manage-learn/project/project-detail/project-detail.page.ts [hunk 3]
```diff
@@ -16,7 +20,7 @@ import { AppHeaderService } from '@app/services';
...
...ation","solutionDetails":{"type":"observation","subType":"school","_id":"5d0a0cf11e724f059a0d8f11","isReusable":false,"externalId":"CRO-2019-TEMPLATE","name":"CRO-2019"},"projectTemplateId":"5fc4f056ed1ae1783770692f","name":"REJNEESH OBSERVATION2","externalId":"REJNEESH-OBSERVATION2","description":"Task-2 Description","updatedAt":"2020-11-30T14:18:46.647Z","createdAt":"2020-11-30T13:23:33.890Z","__v":0,"status":"notStarted","isImportedFromLibrary":true,"lastSync":"2020-11-30T14:18:46.647Z"}],"learningResources":[{"name":"Copy Feature","link":"https://dev.bodh.shikshalokam.org/resources/play/content/do_113059727462957056137","app":"bodh","id":"do_113059727462957056137"}],"deleted":false,"title":"REJNEESH-TEST","description":"improving community library","updatedAt":"2020-12-01T17:20:04.954Z","createdAt":"2020-11-30T13:15:02.824Z","lastDownloadedAt":"2020-12-11T04:10:37.799Z","lastSync":"2020-12-01T17:20:04.953Z","entityId":"5beaa888af0065f0e0a10515","entityName":"Apple School","programId":"5fc4ad7a4f96e8623deacda9","programName":"Project -30-nov","rationale":"","primaryAudience":["teachers","head master"],"_id":"5fc4ff46ed1ae17837706937","_rev":"1-28270c873915239807bd35fde4d4157f"};
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/project-detail/project-detail.page.ts:23]: can you create a common function in utils and return this object from there for now..

Comment (kiranharidas187) [reply src/app/manage-learn/project/project-detail/project-detail.page.ts:23]: For all the pages

File: src/app/manage-learn/project/project-detail/project-detail.page.ts [hunk 4]
```diff
@@ -242,20 +273,20 @@ export class ProjectDetailPage implements OnInit {
   // task and project delete permission.
   async askPermissionToDelete(type, id?) {
     let data;
-    this.translate.get(["MESSAGES.DELETE_CONFIRMATION", "LABELS.CANCEL", "LABELS.SUBMIT"]).subscribe((text) => {
+    this.translate.get(["DELETE_CONFIRMATION", "CANCEL", "BTN_SUBMIT"]).subscribe((text) => {
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/project-detail/project-detail.page.ts:276]: the strings keys should be of structur
FRAGMENT_<LBL/ BTN>_ DELETE_CONFIRMATION


Please go to en.json and search for FRAGMENT_, you can check how it is done and change accordigly

File: src/app/manage-learn/project/project-detail/project-detail.page.ts [hunk 5]
```diff
@@ -242,20 +273,20 @@ export class ProjectDetailPage implements OnInit {
...
       data = text;
     });
     const alert = await this.alert.create({
-      message: data["MESSAGES.DELETE_CONFIRMATION"] + type + "?",
+      message: data["DELETE_CONFIRMATION"] + type + "?",
```
Comment (kiranharidas187) [inline src/app/manage-learn/project/project-detail/project-detail.page.ts:280]: Do the above thing for all the  Translates

File: src/app/manage-learn/project/learning-resources/learning-resources.page.html (hunk 2 above)
Comment (harpreetwalia143) [reply src/app/manage-learn/project/learning-resources/learning-resources.page.html:13]: they are not using it, I checked in other components and pages

**Pull request merged**